tests/empty-definitions.test.ts

```diff
@@ -30,62 +30,8 @@ describe("allow empty definitions", () => {
       },
     };
 
-    expect(await openapiTS(schema as any, { version: 2 })).toBe(`/**
- * This file was auto-generated by openapi-typescript.
- * Do not make direct changes to the file.
- */
-
-export interface paths {
-  "/pet": {
-    post: operations["addPet"];
-  };
-}
-
-export interface operations {
-  addPet: {
-    parameters: {
-      body: {
-        /** Pet object that needs to be added to the store */
-        body: definitions["Pet"];
-      };
-    };
-    responses: {
-      /** Invalid input */
-      405: unknown;
-    };
-  };
-}
-
-export interface external {}
-`);
-
-    expect(await openapiTS(schema as any, { immutableTypes: true, version: 2 })).toBe(`/**
- * This file was auto-generated by openapi-typescript.
- * Do not make direct changes to the file.
- */
-
-export interface paths {
-  readonly "/pet": {
-    readonly post: operations["addPet"];
-  };
-}
-
-export interface operations {
-  readonly addPet: {
-    readonly parameters: {
-      readonly body: {
-        /** Pet object that needs to be added to the store */
-        readonly body: definitions["Pet"];
-      };
-    };
-    readonly responses: {
-      /** Invalid input */
-      readonly 405: unknown;
-    };
-  };
-}
-
-export interface external {}
-`);
+    // We are using snapshots to enforce consistency in string output and indentation
+    expect(await openapiTS(schema as any, { version: 2 })).toMatchSnapshot();
```

Comment:
👍🏻 I’m fine with snapshots here. As you may have guessed I’m partial to having the full-file “snapshots” just so they could be more-easily readable. Also in a project like this where nearly everything is a snapshot, I didn’t want a careless `-u` flag causing a huge regression and liked the added friction of having to edit a file.

But that’s for the full-schema tests; I agree that the inline snapshots here were noisy, and not as useful. I’m OK with this change.